source/windows/bcrypt_aes.c

```diff
@@ -555,120 +536,162 @@ struct aws_symmetric_cipher *aws_aes_cbc_256_new_impl(
     return NULL;
 }
 
-/* the buffer management for this mode is a good deal easier because we don't care about padding.
-   We do care about keeping the final buffer less than a block size til the finalize call so we can
-   turn the auth chaining flag off and compute the GMAC correctly. */
+/*
+ * The buffer management for this mode is a good deal easier because we don't care about padding.
```

Comment:
trivial: Say WHAT it's easier than. I don't know what this is talking about, and it would be nice to know what it's comparing to

Reply:
that was actually an existing comment that was moved around. made it more explicit.